rack-protection/rack-protection.gemspec

```diff
@@ -1,10 +1,9 @@
-$:.unshift File.expand_path("../../rack-protection/lib", __FILE__)
-require "rack/protection/version"
+version = File.read(File.expand_path("../../VERSION", __FILE__)).strip
 
```

Comment:
Ah true, after I commented this I realized it should work if we do the `git` block form.